### ISSUE TITLE
documentation website down.

### ISSUE THREAD

**Issue by mathematicalmichael** [UT-CHG/BET#319]
I looked into why this might be and I think it's because the `makefile` is assuming the existence of `../gh-pages/` which I understand to be the branch with no common commit history, but was held as a folder in a parallel directory. So yeah, that's my operating theory ... a user (me) following instructions as-is in `README` didn't actually push changes to the website. Can't investigate further since Settings doesn't show up for me.

EDIT: I see now the `gh-pages/`folder is generated during `make`

**Comment by smattis** [comment]
I just pushed a change that fixes it. Apparently addresses github pages sites are now case sensitive. That was the problem. We do need to rebuilt the docs, though, which is a little tricky.

**Comment by mathematicalmichael** [comment]
thanks! is the problem with rebuilding them related to the whole "orphan branch" thing? 
I have a bash script that does pushes of a `public` folder of a static site to a `gh-pages` branch, so that may be helpful? [link to bash script on github](https://github.com/mathematicalmichael/slumlords/blob/master/move_site.sh)

**Comment by mathematicalmichael** [comment]
@smattis I don't see that the `gh-pages` branch has been updated at all. Where/how is github hosting the webpage from?? I've dug into the `Makefile` and I suppose my knowledge ends at Sphinx Docs, but from what I can tell, this is just a static-site generator based on docstrings.

From my understanding of how github pages works, it hosts the website from one of two places:
master branch and doc folder OR gh-pages branch.
The latter has a bunch of HTML pages in it, which would make the most sense to me.

So I suppose that when you "pushed a change" you just meant that the settings had been tweaked, because the .rst files in /doc should not be enough for github pages to render a site. Therefore, I think that I was right in my assumption in April about the orphan branch. I have set one of these up before and from memory... rebuilding our docs should go something like this?

- make html (twice?)
- now gh-pages folder exists but is untracked. 
- checkout orphan branch, since folder untracked, it should show up (or yell at you for being untracked, so move the folder, checkout, move it back).
- migrate files to the root directory of the orphan branch, commit, push.

sound about right? If so, I'll try to rebuild the docs myself (and pending your permission push to the orphan branch... or submit a PR for it?). Once #340 is merged, I would very much like to update the docs.
